source/Calamari.AzureAppService.Tests/AzureAppServiceDeployContainerBehaviourFixture.cs

```diff
@@ -18,136 +18,260 @@
 
 namespace Calamari.AzureAppService.Tests
 {
-    [TestFixture]
-    public class AzureAppServiceDeployContainerBehaviorFixture : AppServiceIntegrationTest
+    public class AzureAppServiceDeployContainerBehaviourFixture
     {
-        CalamariVariables newVariables;
-        readonly HttpClient client = new HttpClient();
+        [TestFixture]
+        public class WhenUsingAWindowsAppService : AppServiceIntegrationTest
```

Comment:
This and the linux test are checking the exact same thing, in the same way, just with a different setup, correct?  
Should they share the `Azure[Windows/Linux]ContainerDeploy()` and `Azure[Windows/Linux]ContainerSlotDeploy()` methods in that case?  Generally I'm not a fan of sharing test code, but these seem like they're actually the same tests.  
It's a little unfortunate that the AppServiceIntegrationTest structure is such that you can't use different setups for the same tests.

Reply:
Correct. They have different setups

They could share the code 🤔 I'm probably more inclined to keep the separate for now. I'm not sure we gain _that_ much by sharing the code

Reply:
My main reason for suggesting sharing is that currently it's hard to quickly tell that they're doing the exact same thing.  Adding a comment saying "this is does the same thing as the linux tests with diff setups" would provide a similar outcome.
Up to you though, it's not a huge deal.

Reply:
Added a comment